framework/auth/views.py

```diff
@@ -191,7 +192,7 @@ def auth_logout(redirect_url=None):
 def auth_email_logout(token, user):
     """When a user is adding an email or merging an account, add the email to the user and log them out.
     """
-    redirect_url = web_url_for('auth_login') + '?existing_user={}'.format(escape_html(user.email))
+    redirect_url = web_url_for('auth_login') + '?existing_user={}'.format(urllib.quote_plus(user.email))
```

Comment:
Is this query param still necessary, since "existing user" is now passed as a context var?